src/frontend/Komponenter/Behandling/Brev/BrevmenyVisning.tsx

```diff
@@ -150,7 +150,10 @@ const BrevmenyVisning: React.FC<BrevmenyVisningProps> = ({
         }, {});
     };
 
+    const [generererBrev, settGenerererBrev] = useState(false);
     const genererBrev = () => {
+        if (generererBrev) return;
```

Comment:
Finnes det en risiko her att man ikke får generert det siste man har endret? 
Dvs jeg endrer noe - det lagres
Mens det lagres så endrer jeg noe nytt, men som ikke blir lagret fordi kallet fra første lagringen fortsatt er igang?

Reply:
Ja, det er det godt poeng. Med denne løsningen kan vi risikere at siste endring ikke sendes. 

Forslag til hvordan vi unngår det? Må vi "batche" opp requests i frontend?

Reply:
Vet ikke helt hvordan man burde løse dette ellers, @charliemidtlyng @Assios ?

Reply:
Hva skjer hvis alle bruker debounced? Er det "for tregt" ?

Reply:
Tja, kanskje ikke. Hadde jo løst problemet.